core/src/main/scala/scala/js/exp/TraitsExp.scala

```diff
@@ -35,7 +35,7 @@ trait TraitsExp extends Traits with ProxyExp {
     }
 
     val implClazz = Class.forName(traitClazz.getName + "$class")
-    val parents = traitClazz.getInterfaces.filter(_ != implicitly[Manifest[scala.ScalaObject]].erasure)
+    val parents = traitClazz.getInterfaces.filter(_ != implicitly[Manifest[AnyRef]].runtimeClass)
```

Comment:
To my understanding, the code
```scala
val parents = traitClazz.getInterfaces.filter(_ != implicitly[Manifest[ScalaObject]].runtimeClass)
```
is to filter out the uninteresting `ScalaObject` root interface that is definitely inherited by all classes we are interested in. Since [`ScalaObject`](http://www.scala-lang.org/api/2.10.4/index.html#scala.ScalaObject) has been [removed from the language](https://groups.google.com/forum/#!topic/scala-user/lLRc_NOboLU), it seems that we no longer need this filter. Or do you think we still want to filter out `AnyRef` as an uninteresting root interface?